src/sentry/buffer/base.py

```diff
@@ -38,10 +38,10 @@ def get(
 
     def get_hash(
         self, model: type[models.Model], field: dict[str, models.Model | str | int]
-    ) -> list[dict[str, str]]:
+    ) -> None:
         return
 
-    def get_set(self, key: str) -> list[set[int]]:
+    def get_set(self, key: str) -> None:
```

Comment:
It's a little weird to have a different return type on a base class vs the subclass, maybe we should just return an empty list here and keep the types consistent?